packages/mobile/src/invite/saga.ts

```diff
@@ -284,10 +284,13 @@ export function* redeemInviteSaga({ inviteCode }: RedeemInviteAction) {
     timeout: delay(REDEEM_INVITE_TIMEOUT),
   })
 
-  if (result === true) {
+  if (result.success === true) {
     Logger.debug(TAG, 'Redeem Invite completed successfully')
     yield put(redeemInviteSuccess())
     navigate(Screens.VerificationEducationScreen)
+    // Note: We are ok with this succeeding or failing silently in the background,
+    // user will have another chance to register DEK when sending their first tx
+    yield call(registerAccountDek, result.newAccount)
```

Comment:
Will user be able to verify later if this fails?

Reply:
Consider `yield spawn` here so this saga can close out.

Reply:
Yeah if they arent registered here, then they register their DEK with their first transfer transaction